components/gitpod-db/go/oidc_client_config.go

```diff
@@ -55,11 +55,15 @@ type OIDCSpec struct {
 
 func CreateOIDCClientConfig(ctx context.Context, conn *gorm.DB, cfg OIDCClientConfig) (OIDCClientConfig, error) {
 	if cfg.ID == uuid.Nil {
-		return OIDCClientConfig{}, errors.New("id must be set")
+		return OIDCClientConfig{}, errors.New("ID must be set")
+	}
+
+	if cfg.OrganizationID == uuid.Nil {
+		return OIDCClientConfig{}, errors.New("Organization ID must be set")
 	}
 
 	if cfg.Issuer == "" {
-		return OIDCClientConfig{}, errors.New("issuer must be set")
+		return OIDCClientConfig{}, errors.New("Issuer must be set")
```

Comment:
```suggestion
		return OIDCClientConfig{}, errors.New("issuer must be set")
```
Go errors should start with lowercase, because these are chained together, so they should read as a set of statements like `doesnt work: config was empty: org ID is nil`